op-deployer/pkg/deployer/utils.go

```diff
@@ -40,17 +43,7 @@ func cwd() string {
 }
 
 func DefaultCacheDir() string {
-	var cacheDir string
-
-	homeDir, err := os.UserHomeDir()
-	if err != nil {
-		cacheDir = ".op-deployer/cache"
-		log.Printf("error getting user home directory: %v, using fallback directory: %s\n", err, cacheDir)
-	} else {
-		cacheDir = path.Join(homeDir, ".op-deployer/cache")
-	}
-
-	return cacheDir
+	return flags.DefaultCacheDir()
 }
```

Comment:
Is it necessary to have this wrapper function that does nothing except call another function?